src/WebApi/Controllers/ValuesController.cs

```diff
@@ -28,7 +30,13 @@ public ValuesController(ILogger logger, IServiceRemoting serviceRemoting)
         public async Task<int> Get(int a, int b)
         {
             var uri = new Uri($"{FabricRuntime.GetActivationContext().ApplicationName}/MyStateless");
-            var sum = await _serviceRemoting.CallAsync<IMyService, int>(HttpContext.TraceIdentifier, uri, service => service.CalculateSumAsync(a, b));
+
+            Func<CustomHeaders> customHeadersProvider = () => new CustomHeaders
```

Comment:
Could you please explain why do you use `Func` instead of instance of `CustomHeaders`?

Reply:
I'll take a look, at this point it is not needed to use the Func I guess.